Dictionary/Dictionary.java

```diff
@@ -0,0 +1,15 @@
+
+import java.util.List;
+
+public interface Dictionary {
+
+	void add(KeyValueNode keyValuePair);// adds a word meaning pair in the dictionary

```

Comment:
this should have key and value as parameter. We are creating dictionary, user don't know what is keyValuePair, we are internally using it as keyvaluepair, which needs not to be expose to user.